BEST_PRACTICES.md

```diff
@@ -123,8 +136,8 @@ Image names should be named consistently to preserve organization and developer
 
 **For example:**
 
-* `refresh-bar-button-item` / `refresh-bar-button-item@2x` and `refresh-bar-button-item-selected` / `refresh-bar-button-item-selected@2x`
```

Comment:
Could be, the only issue is that using images is much more efficient than drawing with code.